yeetd/main.swift

```diff
@@ -8,22 +8,28 @@
 import Foundation
 import OSLog
 
+UserDefaults.standard.setValue(true, forKey: "killapsd")
```

Comment:
I kinda lean towards removing this - I think `apsd` should never be the default TBH.